src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -172,3 +177,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.custom_command('create', 'pnp_create_key')
         g.custom_command('delete', 'pnp_delete_key')
         g.custom_command('update', 'pnp_update_key')
+
+    with self.command_group('iot central app', iotcentral_sdk, client_factory=iot_central_service_factory) as g:
+        g.custom_command('create', 'iotcentral_app_create')
+        g.custom_command('list', 'iotcentral_app_list')
+        g.custom_command('show', 'iotcentral_app_get')
```

Comment:
Please use `custom_show_command` so the command will return an exit code of 3 in case of 404.

Reply:
removed